SpecialOperations2.Altis/sos/vehicle/functions/fn_activateServicePoint.sqf

```diff
@@ -19,9 +19,10 @@ private ["_vehicle", "_actionName", "_actionId"];
 _vehicle = vehicle player;
 
 //Checks for damage, fuel and touching ground (cant move people out of flying choppers, also ammo later on)
+    
 
 if (((damage _vehicle) > 0) or ((fuel _vehicle) < 1)) then {
-    _actionId = _vehicle addAction [
+    _actionId = driver _vehicle addAction [
```

Comment:
Its completely safe because like i said, you have to be the driver so you can assume the player is the driver at this point - like you do in 'deactivateServicePoint'!